### PR TITLE
Build ops container

### PR DESCRIPTION
**Pull request by jesse-dds**
- Adds a flag `build_and_push_ops_image` to trigger building and storing ops image in the registry
- Adds environment variables `AZURE_SUBSCRIPTION_OPS` and `AZURE_REGISTRY_OPS`
- Breaks `azlogin` into `install_azure_cli` and `log_into_ops_registry` and `log_into_app_registry`

### CODE REVIEW

**Review by mikepw — APPROVED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -352,11 +366,64 @@ jobs:
...
+          shell: /bin/sh
+          command: |
+            az login \
+              --service-principal \
+              --tenant << pipeline.parameters.tenant_id >> \
+              --password << pipeline.parameters.sp_password >> \
+              --username << pipeline.parameters.sp >>
+            az account set -s << pipeline.parameters.subscription_id >>
+            echo "Successfully logged in to Azure CLI."
+            az acr login --name << pipeline.parameters.ops_registry >> | grep "Succeeded"
```
Comment (dandds) [inline .circleci/config.yml:399]: Could lines 378 to here just reuse the `azlogin` command?

**Review by dandds — COMMENTED** [review]
Looks good, just one quick question.

File: .circleci/config.yml (hunk 1 above)
Comment (jesse-dds) [reply .circleci/config.yml:399]: Yeah actually, just need to parameterize the registry url

**Review by dandds — APPROVED** [review]
👍

**Pull request merged**